src/redist/targets/BundledTemplates.targets

```diff
@@ -87,25 +115,32 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageDownload Include="@(Bundled22Templates);@(Bundled21Templates)" />
+    <PackageDownload Include="@(Bundled30Templates);@(Bundled22Templates);@(Bundled21Templates)" />
   </ItemGroup>
 
   <!-- Restore bundled templates via PackageReference -->
   <ItemGroup>
-    <PackageReference Include="@(Bundled30Templates)" />
+    <PackageReference Include="@(Bundled31Templates)" />
```

Comment:
Is there a reason we use `PackageDownload` for older templates and `PackageReference` for the current ones?  Should they all just use `PackageDownload`?

Reply:
@johnbeisner suggested there was something that didn't work, but I can't see what. I'm leaving well enough alone on this to unblock code flow. @johnbeisner has been thinking of doing some reworking of all of this and I believe he even started on it, so I'll avoid separate cleanup that may just add more conflicts.